sources/packages/api/src/route-controllers/institution/institution.aest.controller.ts

```diff
@@ -38,10 +53,10 @@ export class InstitutionAESTController extends BaseController {
    * @param institutionId
    * @returns InstitutionDetailDTO.
    */
-  @Get("/:institutionId")
```

Comment:
oh.. will this also work?

Reply:
Yes it works. I assume you are asking about the leading slash. Btw. It got carried from old code.

Reply:
ya